src/api/bridgedDeposit.js

```diff
@@ -54,12 +53,9 @@ module.exports = async (dvf, data, nonce, signature, txHashCb) => {
   }
   const [transactionHashPromise, transactionHashCb] = createPromiseAndCallbackFn(txHashCb)
 
-  if (dvf.dvfStarkProvider && dvf.dvfStarkProvider.getWalletType() === 'LEDGER') {
-    await dvf.token.provideContractData(null, tx.tokenAddress, tx.quantum)
```

Comment:
I assume this is fully deprecated and we no longer need contract data for ledger

Reply:
I added it at a time I was only able to test with Metamask.
Testing with Ledger on Matic Mumbai, the transaction succeeds without this step